components/nexus-component-source/src/main/java/org/sonatype/nexus/component/source/config/ComponentSourceRegistryInitializedEvent.java

```diff
@@ -21,11 +20,8 @@
  * @since 3.0
  */
 public class ComponentSourceRegistryInitializedEvent
-    extends AbstractEvent<ComponentSourceRegistry>
 {
-  public ComponentSourceRegistryInitializedEvent(
-      final ComponentSourceRegistry component)
+  public ComponentSourceRegistryInitializedEvent(final ComponentSourceRegistry component)
   {
-    super(component);
```

Comment:
If we're no longer storing or doing anything with `component` does it make sense to keep this constructor?